app/src/main/res/color/overlay_button_border.xml

```diff
@@ -5,6 +5,7 @@
 
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
     <item android:state_pressed="true" android:color="@color/button_pressed" />
-    <item android:state_focused="true" android:color="@color/button_focused" />
+    <item android:state_focused="true" android:color="@color/tv_blue50" />
```

Comment:
Same, just use these colors directly from the photon colors without wrapping/aliasing them.